README.md

```diff
@@ -1,18 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# TODO LIST
 
-> One paragraph statement about the project.
+> "TODO List is simple task organizer written in javascript.
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html/CSS
+- Javascript
+- Webpacks
 
```

Comment:
- [ ]  Please your project is missing the following requirements;

![req](https://user-images.githubusercontent.com/65852621/144721215-4eef4c64-ec6a-4116-aa71-c91068e309f6.png)
Kindly implement them and resubmit for review